shipLHC/AdvTarget.cxx

```diff
@@ -51,7 +51,7 @@ AdvTarget::AdvTarget()
     : FairDetector("AdvTarget", "", kTRUE)
     , fTrackID(-1)
     , fVolumeID(-1)
-    , fPos()
+    , EntryPoint()
```

Comment:
for style: fEntryPoint in place of just EntryPoint maybe?
same for the ExitPoint

Reply:
I'm not a fan of the prefix-f. happy to debate whether this should be snake_case, PascalCase etc., but I think we should really avoid these 20th century conventions.

Reply:
that is the thing, the style philosophy changes with time, but the base classes don't as much.
Since the advsnd hit classes inherit from FairMCPoint, I'd keel the convention of the latter.
But we already break it with the pdgCode.. 
Then maybe use the camel case as done for pdgCode and the f-ed(no pun intended) members?
I don't insist at all.

Reply:
I think generally modern C++ tends to use snake_case for variable names, including member variables.

For member variables, I am aware of the following conventions:
* Google: member_variable_
* Also common: m_member_variable
* [ROOT (actually Taligent)](https://root.cern/contribute/coding_conventions/): fMemberVariable
* member_variable (just like a normal variable)

I'm not a fan of the Taligent way, but indeed, consistency with ROOT/FairRoot is a big pro.

Reply:
Probably I would suggest following the [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#nl-naming-and-layout-suggestions).